src/assets/HTMLAsset.js

```diff
@@ -45,6 +45,10 @@ class HTMLAsset extends Asset {
       if (node.attrs) {
         for (let attr in node.attrs) {
           let elements = ATTRS[attr];
+          // link is not a file
+          if (node.tag === 'a' && node.attrs[attr].lastIndexOf('.') < 1) {
```

Comment:
Maybe we should just ignore all absolute paths within HTML instead? so just check if the first character is `/`?

Reply:
Wouldn't this also break any absolute path require, (that has already been reported as a bug)?

Reply:
Not sure why you'd want to require an absolute path so this might be ok.